<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.5"/>
<title>Affymetrix Power Tools: MANUAL: %Ps_Classification (2.0.1)</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<!-- end header part -->
<!-- Generated by Doxygen 1.8.5 -->
  <div id="navrow1" class="tabs">
    <ul class="tablist">
             <li><a href="index.html"><span>Main&nbsp;Page</span></a></li>
             <li><a href="CHANGE.html"><span>Change&nbsp;Log</span></a></li>
             <li><a href="VIGNETTE.html"><span>Vignettes</span></a></li>
             <li><a href="FAQ.html"><span>FAQs</span></a></li>
             <li><a href="LICENSE.html"><span>License</span></a></li>
             <li><a href="FILE-FORMATS.html"><span>File Formats</span></a></li>
             <li><a href="PLATFORMS.html"><span>Platforms</span></a></li>
    </ul>
  </div>
</div><!-- top -->
<div class="header">
  <div class="headertitle">
<h1>MANUAL: Ps_Classification (2.0.1) </h1>  </div>
</div><!--header-->
<div class="contents">
<div class="textblock"><h2>Contents </h2>
<ul>
<li><a href="#intro">Introduction.</a></li>
<li><a href="#quickstart">Quick Start</a></li>
<li><a href="#options">Options</a></li>
</ul>
<p><a class="anchor" id="intro"></a> </p>
<h2>Introduction </h2>
<p><b>Ps_Classification</b> reads in the SNP QC metrics table generated by <a href="Ps_Metrics.html">Ps_Metrics</a> and performs SNP classification based on customizable criteria, including call rate, FLD, HetSO, and HomRO. It classifies SNPs/probesets into seven major categories: "PolyHighResolution","NoMinorHom", "MonoHighResolution", "OffTargetVariant", "CallRateBelowThreshold", "Other", and "Hemizygous". <code>Ps_Classification</code> also identifies the best probeset per SNP. More details describing the use of Ps_Classification can be found in the <a href="http://media.affymetrix.com/support/developer/downloads/Tools/SNPolisher_User_Guide.pdf">SNPolisher User Guide</a> available on Affymetrix.com.</p>
<p><a class="anchor" id="quickstart"></a> </p>
<h2>Quick Start </h2>
<p>To run <code>Ps_Classification</code> on a metrics file <code>metrics.txt</code> for Human: </p>
<pre class="fragment">Ps_Classification --species-type human --metrics-file metrics.txt --output-dir ./output
</pre><p><a class="anchor" id="details"></a> </p>
<h2>Details </h2>
<p>The main parameters accepted by <code>Ps_Classification</code> are:</p>
<ul>
<li><b>metrics-file</b> - The name of the file containing output generated by Ps_Metrics.</li>
<li><b>ps2snp-file</b> (Optional) - The name of a file containing the mapping of probeset IDs to SNP IDs. The column names should be "probeset_id" and "snpid". The user must supply this file when classifying de novo SNPs or any SNPs that have more than one probeset.</li>
<li><b>output-dir</b> - Output directory for results files: defaults to current directory.</li>
<li><b>species-type</b> - The type of organism genotyped. The current options are "human", non-human "diploid", and "polyploid".</li>
<li><b>converted</b> - Flag indicating whether to output a list of converted SNPs to a file <code>converted.ps</code>: default is FALSE.</li>
<li><b>cr-cutoff</b> - Threshold for call rate. If not specified, the default for human is 95 and for diploid and polyploid is 97.</li>
<li><b>fld-cutoff</b> - Threshold for FLD: default=3.6.</li>
<li><b>het-so-cutoff</b> - Threshold for HetSO: default=-0.1.</li>
<li><b>het-so-otv-cutoff</b> - Threshold for OTV detection: default=-0.3.</li>
<li><b>hom-ro-1-cutoff</b> - Threshold for HomRO when a SNP has one genotypes: default=0.6.</li>
<li><b>hom-ro-2-cutoff</b> - Threshold for HomRO when a SNP has two genotypes: default=0.3.</li>
<li><b>hom-ro-3-cutoff</b> - Threshold for HomRO when a SNP has three genotypes: default=-0.9.</li>
<li><b>hom-ro</b> - Flag indicating whether the metric HomRO is used in classification: default is TRUE.</li>
<li><b>hom-het</b> - Flag indicating whether the metric HomHet is used in classification. The HomHet metric identifies two-cluster SNPs/probesets with one homozygote cluster and one heterozygote cluster. This checks if the minor homozygote cluster is missing, which is unreasonable for highly inbred species (e.g. wheat). This metric should be turned on when classifying SNPs/probesets in highly inbred species: default is TRUE.</li>
<li><b>num-minor-allele-cutoff</b> - Threshold for the number of minor alleles: default=2.</li>
<li><b>priority-order</b> - When performing probeset selection, the best probeset is selected according to the priority order of probeset conversion types. Valid values are sequences of the strings <em>"PolyHighResolution"</em>, <em>"NoMinorHom"</em>, <em>"OTV"</em>, <em>"MonoHighResolution"</em> and <em>"CallRateBelowThreshold"</em>, separated by commas, with each string occuring exactly once: default <code>'PolyHighResolution,NoMinorHom,OTV,MonoHighResolution,CallRateBelowThreshold'</code></li>
</ul>
<p><code>Ps_Classification</code> reads in the SNP QC metrics table generated by <a href="Ps_Metrics.html">Ps_Metrics</a> and performs SNP classification based on the customizable criteria. It classifies SNPs/probesets into seven major categories: "PolyHighResolution","NoMinorHom", "MonoHighResolution","OTV", "CallRateBelowThreshold", "Other", and "Hemizygous". The metric <code>hom-ro</code> shoud be turned off for organisms where the homozygote clusters may center at zero in the contrast space, particularly polyploid species. If a <code>ps2snp-file</code> is provided, then the best performance probeset will be selected to represent SNPs with multiple probesets. There are seven output files, one for each category, containing lists of probeset IDs belonging to each category. Each file contains a list of probeset IDs belonging to the category. There is also a summary output file.</p>
<p><a class="anchor" id="options"></a> </p>
<h2>Options: </h2>
<pre class="fragment">
   'Ps_Classification'
   2.0.1 svn://svn.affymetrix.com/Applications/SNPPolisher/branches/SP_2_0_1_BRANCH@17261:17262(1276) 2014-06-19-16:37


USAGE:
     Ps_Classification --species-type [human|polyploid|diploid] --metrics-file &lt;FILE&gt; --output-dir &lt;DIR&gt; [OPTIONS]

USER HELP ----------

  --version (flag) = Display the version info for this program.

  --user-help (flag) = Display help intended for end-users.
    Aliases: --help, -h

  --log-file (string8) = The name of the log file.  Defaults to 'PROGRAMNAME-NUMBER.log'.
    Aliases: --lf

  --console-add-select (string8) = Add selectors for console messages.
    e.g. to include all debug messages: --console-add-select debug
         to include all messages of level 3 or higher: --console-add-select '*:3'


  --console-add-neg-select (string8) = Add selectors to be excluded from console messages.
    e.g. to exclude all warning messages and errors summary: --console-add-neg-select WARNING,summary


  --arg-file (filename) = Read arguments from this file.
    File should be a valid XML file, and should contain elements of the form
    &lt;Parameter name="PARAMETER-NAME" analysis="NODE-NAME" currentValue="VALUE" /&gt;.

  --check-param (flag) = Stop the program after the command parameters have been checked.

  --metrics-file (filename,required) = Full path name for Metrics file generated by Ps_Metrics.

  --ps2snp-file (filename) = Full path name of file mapping probeset IDs to SNP IDs (Recommended).

  --output-dir (dirname) = Path to output directory.

  --species-type (string8) = Type of organism to be genotyped. Valid options are 'human','diploid' or 'polyploid'.
    default value: 'human'

  --converted (bool) = If true output a list of converted SNPs to file converted.ps.

  --cr-cutoff (float64) = Threshold for call rate. Default is 95.0 for Human, 97.0 for Diploid or Polyploid.
    default value: '95.000000'

  --fld-cutoff (float64) = Threshold for FLD
    default value: '3.600000'

  --het-so-cutoff (float64) = Threshold for HetSO
    default value: '-0.100000'

  --het-so-otv-cutoff (float64) = Threshold for HetSO OTV
    default value: '-0.300000'

  --hom-ro-1-cutoff (float64) = Threshold for HomRO when SNP has 1 genotype
    default value: '0.600000'

  --hom-ro-2-cutoff (float64) = Threshold for HomRO when SNP has 2 genotype
    default value: '0.300000'

  --hom-ro-3-cutoff (float64) = Threshold for HomRO when SNP has 3 genotype
    default value: '-0.900000'

  --hom-ro (flag) = True if HomRO metric to be used in classification 

  --hom-het (flag) = True if Hom Het metric to be used in classification

  --num-minor-allele-cutoff (float64) = Threshold for number of minor alleles
    default value: '2.000000'

  --priority-order (string8) = Priority order of probeset conversion types when performing probeset selection.
    default value: 'PolyHighResolution,NoMinorHom,OTV,MonoHighResolution,CallRateBelowThreshold'

  --@help       = print developer help

</pre> </div></div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated for Affymetrix Power Tools by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.5
</small></address>
</body>
</html>
