<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<title>Affymetrix Power Tools: VIGNETTES: CytoScan750K Array</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<link href="doxygen.css" rel="stylesheet" type="text/css"/>
</head>
<body>
<!-- Generated by Doxygen 1.7.1 -->
<div class="navigation" id="top">
  <div class="tabs">
    <ul class="tablist">
             <li><a href="index.html"><span>Main&nbsp;Page</span></a></li>
             <li><a href="CHANGE.html"><span>Change&nbsp;Log</span></a></li>
             <li><a href="VIGNETTE.html"><span>Vignettes</span></a></li>
             <li><a href="FAQ.html"><span>FAQs</span></a></li>
             <li><a href="LICENSE.html"><span>License</span></a></li>
             <li><a href="FILE-FORMATS.html"><span>File Formats</span></a></li>
             <li><a href="PLATFORMS.html"><span>Platforms</span></a></li>
    </ul>
  </div>
</div>
<div class="header">
  <div class="headertitle">
<h1>VIGNETTES: CytoScan750K Array </h1>  </div>
</div>
<div class="contents">
<dl class="date"><dt><b>Date:</b></dt><dd>2012-01-26</dd></dl>
<h2><a class="anchor" id="content"></a>
Contents</h2>
<ul>
<li><a href="#intro">Introduction</a></li>
<li><a href="#libraryfiles">Library Files</a></li>
<li><a href="#ref">Command line example for reference creation:</a></li>
<li><a href="#ssa">Command line example for single sample mode processing</a></li>
</ul>
<p><a class="anchor" id="intro"></a> </p>
<h2><a class="anchor" id="intro"></a>
Introduction</h2>
<p>This vignette describes the process for using apt-copynumber-cyto to process CytoScan750K arrays under reference creation mode and single sample mode.</p>
<p>apt-copynumber-cyto implements two distinct workflows, a reference workflow that uses as a a set of CEL files that are input to outputs one or more reference files including copynumber and SNP information; and conceptually, a single-sample workflow that compares each CEL file to a pre-computed reference for analysis. For efficiency of computation the "single-sample workflow" operates on a set of input CEL files at a time, but the output for any CEL file is unaffected by any of the other CEL files.</p>
<p><a class="anchor" id="libraryfiles"></a> </p>
<h2><a class="anchor" id="libraryfiles"></a>
Library Files</h2>
<p>The following library files are required for analysis of CytoScan750K array data.</p>
<div class="fragment"><pre class="fragment">
CytoScan750K_Array.na32.1.annot.db:     NetAffx Annotation database file. It is a SQLite 3.x database.

CytoScan750K_Array.chrXprobes:          An ASCII file that contains probe_id (1-based) of
                                        probes on chrX. Used for copy number probe chrX/Y ratio gender calling.

CytoScan750K_Array.chrYprobes:          An ASCII file that contains probe_id (1-based) of
                                        probes on chrY as column 1 and probeset_id as column 2. Probe_id is used for
                                        copy number probe chrX/Y ratio gender calling and probeset_id is used for
                                        hasY gender call and Y-Gender call.

CytoScan750K_Array.r1.qca:              An ASCII file that defines the parameters for the calculations of qc-call-rate
                                        and contrast-qc-rand that are used in the geno-qc analysis.

CytoScan750K_Array.r1.qcc:              An ASCII file that defines the probesets that are used by the geno-qc analysis

CytoScan750K_Array.cdf:                 Chip definition file (binary file).

CytoScan750K_Array.probe_tab:           An ASCII file that contains probe sequence information. (Reference creation only).

CytoScan750K_Array.hg19.refcovar:       An hdf5 format file that contains covariate information for each probeset, where
                                        the first column is the probeset name and subsequent column(s) are covariates in float.
                                        (Reference creation only).

CytoScan750K_Array.snplist.txt:         An ASCII file that contains SNP probeset_id that are used to compute processed SNPQC and Raw SNPQC.
                                        The first row of the file must be "probeset_id". (Single sample mode processing only).

CytoScan750K_Array.na32.1.v1.REF_MODEL: Affymetrix copy number reference generated from a set of hapmaps and normal blood samples (hdf5 format file).
</pre></div><p><a class="anchor" id="ref"></a> </p>
<h2><a class="anchor" id="ref"></a>
Command line example for reference creation:</h2>
<p>NOTE: This command needs to have the comments removed to be runnable, as they break the unix shell syntax. The comments and additional spaces are to clarify the options being used. The arguments which are joined with a "."s should be joined without spaces.</p>
<div class="fragment"><pre class="fragment">
apt-copynumber-cyto -v 4 \             #verbose output level in the log file ranging from 1 (least details) to 4 (most details)
	--cyto2 false \                        #true: Cytogenetics_Array; false: CytoScanHD_Array,CytoScan750K_Array
	--force false \                        #true: Disable various checks including chip types; false: otherwise.
	--keep-intermediate-data false \       #true: dump out intermediate data during apt processing; false: otherwise.
	--doDualNormalization true \           #true: quantile normalize CN and SNP separately; false: do not quantile normalize CN and SNP separately
	--adapter-type-normalization false \   #true: perform adapter type normalization; false: otherwise. Should be always set to "false" for CytoScan750K_Array as the adjustment has been incorporated in "signal-adjustment-covariates"
	--probe-file CytoScan750K_Array.probe_tab \ #specify the probe sequence file
	--run-geno-qc true \                   #true: Run the GenoQC engine; false: otherwise
	--qca-file CytoScan750K_Array.r1.qca \ #if run-geno-qc is set to be true, it is required to provide this file
	--qcc-file CytoScan750K_Array.r1.qcc \ #if run-geno-qc is set to be true, it is required to provide this file
	--text-output false \                  #true: Output data in ASCII text format in addition to calvin format; false: otherwise
	--cnchp-output false \                 #true: report CNCHP file(s); false: otherwise. Either cnchp-output or cychp-output has to be true
	--cychp-output true \                  #true: report CYCHP file(s); false: otherwise. Either cnchp-output or cychp-output has to be true
	--reference-output CnNewRef.REF_MODEL \   #specify name of the new copy number reference file with file extension "REF_MODEL"
	--cdf-file CytoScan750K_Array.cdf \    #specify the chp definition file
	--chrX-probes CytoScan750K_Array.chrXprobes \ #specify the chrX probe file
	--chrY-probes CytoScan750K_Array.chrYprobes \ #specify the chrY probe file
	--annotation-file CytoScan750K_Array.na32.1.annot.db \  #specify the NetAffx database file
	--male-gender-ratio-cutoff 1.5 \       #male gender call threshold for chrX/Y ratio gender calling
	--female-gender-ratio-cutoff 0.9 \     #female gender call threshold for chrX/Y ratio gender calling
	--xx-cutoff 0.65 \                     #lower limit of the gender call threshold for hasXX gender call
	--xx-cutoff-high 0.95 \                #upper limit of the gender call threshold for hasXX gender call
	--y-cutoff 0.58 \                      #gender call threshold for hasY gender call
	--temp-dir temp_dir \                  #specify the temporary directory for intermediate data from reference generation
	--out-dir output_dir \                 #specify the output directory
	--covariates-file  CytoScan750K_Array.hg19.refcovar \ #specify the covariate file that contains probeset-based covariate(s) that doesn't exist in the annotation db file
	--signal-adjustment-covariates covariate-signal-adjuster.\  #covariate-signal adjuster
		order=fragment-adapter-type,fragment-length.\       #the covariates to be used in the order in which they are to be applied
		bin-type=discrete,equally-populated.\               #ordered list corresponding to each covariate controls whether the covariate is treated directly as discrete variable, or as a continuous variable (equally-populated or equally-spaced)
		bin-count=0,100 \                                   #ordered list corresponding to each covariate controls how a continuous variable is discretized into bins. Must be 0 for discrete covariates
	--lr-adjustment-covariates covariate-lr-adjuster.\          #covariate log2 ratios adjuster
		order=SuperGC,median-signal,marker-class.\          #the covariates to be used in the order in which they are to be applied
		bin-type=discrete,equally-populated,discrete.\      #ordered list correspoinding to each covariate controls whether the covariate is treated directly as discrete variable, or as a continuous variable (equally-populated or equally-spaced)
		bin-count=0,100,0\                                  #ordered list corresponding to each covariate controls how a continuous variable is discretized into bins. Must be 0 for discrete covariates
		.iqr-scaling=on,on,on.\                             #ordered list corresponding to each covariate on scales IQR for each bin to be equal
		subtract-from-XY=on,off,on \                        #ordered list corresponding to each covariate. “on” applies covariate adjustment to ChrX,ChrY markers.
	--wave-correction-reference-method wave-correction-reference-method.\ #estimate waves using the reference samples
		trim=2.0.\                                          #trim parameter for absolute adjusted log2 ratios after each wave
		percentile=0.75.\                                   #percentile to use for each probeset in finding wave. E.g., 0.75 means 75-th percentile.
		wave-count=6.\                                      #number of waves to calculate
		demean=false \                                      #true: demean prior to finding each wave; false: otherwise
	--local-gc-background-correction-reference-method none \    #should be always set to "none" for CytoScan750K_Array
	--local-gc-background-intensity-adjustment-method none \    #should be always set to "none" for CytoScan750K_Array
	--image-correction-intensity-adjustment-method none \       #should be always set to "none" for CytoScan750K_Array
	--allele-peaks-adjustment-covariates covariate-allele-peaks-adjuster.\ #covariate allelic difference adjuster
		order=local-gc.\                                    #the covariates to be used in the order in which they are to be applied
		bin-type=equally-populated.\                        #ordered list corresponding to each covariate controls whether the covariate is treated directly as discrete variable, or as a continuous variable (equally-populated or equally-spaced)
		bin-count=100.\                                     #ordered list corresponding to each covariate controls how a continuous variable is discretized into bins. Must be 0 for discrete covariates
		coarse-allele-peak-adjustment=on.\                  #if it is 'on' then turn on coarse allelic peak adjustment, and 'off' otherwise
		coarse-allele-peak-adjustment-step=50.\             #step size for another density estimation across the genome
		coarse-allele-peak-adjustment-window=200.\          #number of SNPs used by the kernel density estimator
		coarse-allele-peak-adjustment-point-count=128.\     #number of output grids for the kernel density estimator
		coarse-allele-peak-adjustment-bandwidth=0.25.\      #adjustment to the data-adaptive bandwidth for the kernel density estimator. Must be 0 &lt; bandwidth &lt;= 1.
		coarse-allele-peak-adjustment-cutoff=0.05.\         #parameter to remove density peaks that are likely due to noise
		coarse-allele-peak-adjustment-clean-threshold=0.35.\#controls with noisy markers are removed from the visualization because they are indeterminately far from adjacent peaks.
		coarse-allele-peak-adjustment-outlier-trim=3.0.\    #trim input allelic differences to be 3 or -3 if they go beyond those limits.
		master-peaks-point-count=513.\                      #number of output grids for the overall density estimator
		master-peaks-bandwidth=0.45.\                       #adjustment to the data-adaptive bandwidth for the per-covariate bin density estimator. Must be 0 &lt; bandwidth &lt;= 1 .
		covariate-bin-peaks-point-count=257.\               #number of output grids for per-covariate bin density estimator
		covariate-bin-peaks-bandwidth=0.45.\                #adjustment to the data-adaptive bandwidth for the per-covariate bin density estimator. Must be 0 &lt; bandwidth &lt;= 1 .
		kernel-function-selection=gaussian \                #The kernel choice for the density estimator. Currently only accept Gaussian kernel.
	--cel-files CELFileList.txt                                 #an ASCII file that contains the full path of input CEL files for reference creation, where the first row of the file is "cel_files" and each subsequent row corresponds to each CEL file
</pre></div><p><a class="anchor" id="ssa"></a> </p>
<h2><a class="anchor" id="ssa"></a>
Command line example for single sample mode processing</h2>
<div class="fragment"><pre class="fragment">
apt-copynumber-cyto -v 4 \                                  #verbose output level in the log file ranging from 1 (least details) to 4 (most details)
	--cyto2 false \                                             #true: Cytogenetics_Array; false: CytoScanHD_Array, CytoScan750K_Array
	--doDualNormalization true \                                #true: quantile normalize CN and SNP separately; false: do not quantile normalize CN and SNP separately
	--keep-intermediate-data false \                            #true: dump out intermediate data during apt processing; false: otherwise.
	--run-geno-qc true \                                        #true: Run the GenoQC engine; false: otherwise
	--qca-file CytoScan750K_Array.r1.qca \                      #if run-geno-qc is set to be true, it is required to provide this file
	--qcc-file CytoScan750K_Array.r1.qcc \                      #if run-geno-qc is set to be true, it is required to provide this file
	--snp-qc-use-contrast true \                                #only used by SNP6 or Cytogenetics_Array, will be ignored by CytoScan750K_Array
	--snp-qc-snp-list CytoScan750K_Array.snplist.txt \          #specify the snp list to compute the processed SNPQC and Raw SNPQC. If it is not specified, apt uses all SNPs.
	--force false \                                             #true: Disable various checks including chip types; false: otherwise.
	--adapter-type-normalization false \                        #true: perform adapter type normalization; false: otherwise. Should be always set to "false" for CytoScanHD_Array as the adjustment has been incorporated in "signal-adjustment-covariates"
	--text-output false \                                       #true: Output data in ASCII text format in addition to calvin format; false: otherwise
	--cnchp-output false \                                      #true: report CNCHP file(s); false: otherwise. Either cnchp-output or cychp-output has to be true
	--cychp-output true \                                       #true: report CYCHP file(s); false: otherwise. Either cnchp-output or cychp-output has to be true
	--set-analysis-name cy750k \                                #Analysis name to use as prefix for output files. The official name for CytoScan750K_Array is cy750k.
	--cdf-file CytoScan750K_Array.cdf \                         #specify the chp definition file
	--chrX-probes CytoScan750K_Array.chrXprobes \               #specify the chrX probe file
	--chrY-probes CytoScan750K_Array.chrYprobes \               #specify the chrY probe file
	--annotation-file CytoScan750K_Array.na32.1.annot.db \      #specify the NetAffx database file
	--reference-input CytoScan750K_Array.na32.1.v1.REF_MODEL \  #specify the copy number reference to use
	--out-dir out_dir \                                         #specify the output directory
	--male-gender-ratio-cutoff 1.5 \                            #male gender call threshold for chrX/Y ratio gender calling
	--female-gender-ratio-cutoff 0.9 \                          #female gender call threshold for chrX/Y ratio gender calling
	--xx-cutoff 0.65 \                                          #lower limit of the gender call threshold for hasXX gender call
	--xx-cutoff-high 0.95 \                                     #upper limit of the gender call threshold for hasXX gender call
	--y-cutoff 0.58 \                                           #gender call threshold for hasY gender call
	--local-gc-background-intensity-adjustment-method none \    #should be always set to "none" for CytoScan750K_Array
	--image-correction-intensity-adjustment-method none \       #should be always set to "none" for CytoScan750K_Array
	--wave-correction-log2ratio-adjustment-method wave-correction-log2ratio-adjustment-method\ #apply wave correction
		.bandwidth=101.\                                    #used in non-parametric wave smoothing
		bin-count=25.\                                      #used in non-parametric wave smoothing
		wave-count=6.\                                      #must be less than or equal to the number of waves specified in wave-correction-reference-method
		wave-smooth=true \                                  #true: apply additional non-parametric wave smoothing; false: otherwise.
	--cn-calibrate-parameters calibrated-log2ratios.\           #specify parameters to compute calibrated log2 ratios
		alpha-cn-calibrate=0.564278.\                       #calibrated alpha for autosomes
		alpha-X-cn-calibrate=0.619453.\                     #calibrated alpha for Chr X
		alpha-Y-cn-calibrate=0.494620.\                     #calibrated alpha for Chr Y
		beta-cn-calibrate=1.\                               #calibrated beta for autosomes
		beta-X-cn-calibrate=1.\                             #calibrated beta for Chr X
		beta-Y-cn-calibrate=1 \                             #calibrated beta for Chr Y
	--analysis genotype \                                       #generate genotype calls for genotypable SNPs and the genotype calls are used by LOH analysis module. Should always specify this for CytoScan750K_Array.
	--analysis log2-ratio.\                                     #log2 ratio calculation analysis module
		gc-correction=false.\                               #true: apply gc-correction; false: otherwise. Should aways set it to be false for CytoScanHD_Array as it has done by covariate adjuster module.
		median-autosome-median-normalization=true.\         #true: apply median autosome median correction; false: otherwise.
		median-smooth-marker-count=5 \                      #running median marker count for weighted log2 ratio. It is ignored by CytoScanHD_Array or CytoScan750K_Array, only for Cytogenetics_Array.
	--log2ratio-adjustment-method log2ratio-adjustment-method-high-pass-filter.\ #high pass filter log2 ratio adjustment analysis module
		use=true \                                          #true: use this module for the analysis; false: otherwise.
	--analysis allelic-difference-CytoScan.\                    #allelic peak calculation module
		outlier-trim=3.0.\                                  #trim input allelic differences to be 3 or -3 if they go beyond those limits.
		step=20.\                                           #step size for another density estimation across the genome
		window=100.\                                        #number of SNPs used by the kernel density estimator
		point-count=129.\                                   #number of output grids for the kernel density estimator
		bandwidth=0.25.\                                    #adjustment to the data-adaptive bandwidth for the kernel density estimator. Must be 0 &lt; bandwidth &lt;= 1.
		cutoff=0.05.\                                       #parameter to remove density peaks that are likely due to noise
		clean-threshold=0.35.\                              #controls with noisy markers are removed from the visualization because they are indeterminately far from adjacent peaks.
		symmetry=true \                                     #true: the data is mirrored about the X axis before fitting the density curves; false: otherwise
	--analysis kernel-smooth.\                                  #apply kernel smooth to the log2 ratios
		sigma_span=50 \                                     #bandwidth parameter for Gaussian smooth
	--analysis cn-cyto2.\                                       #Hidden Markove Model (HMM) analysis module
		hmmCN_state=\'0,1,2,3,4\'.\                         #supported CN states for autosome
		hmmCN_mu=\'-2,-0.5,0,0.33,0.54\'\                   #HMM state means for autosome
		.hmmCN_sigma=\'0.3,0.3,0.3,0.3,0.3\'\               #HMM state sigmas for autosome
		.hmmCN_state-X=\'0,1,2,3,4\'.\                      #supported CN states for Chr X
		hmmCN_mu-X=\'-2,-0.55,0,0.36,0.57\'\                #HMM state means for Chr X
		.hmmCN_sigma-X=\'0.3,0.3,0.3,0.3,0.3\'\             #HMM state sigmas for Chr X
		.hmmCN_state-Y=\'0,1,2,3,4\'.\                      #supported CN states for Chr Y
		hmmCN_mu-Y=\'-1.4,-0.5,0,0.3,0.51\'\                #HMM state means for Chr Y
		.hmmCN_sigma-Y=\'0.3,0.3,0.3,0.3,0.3\'\             #HMM state sigmas for Chr Y
		.diagonal-weight-Y=0.995.\                          #transition probability matrix diagnal for Chr Y
		mapd-weight-Y=0.2\                                  #per sample adjustment parameter to HMM state sigmas using MAPD for Chr Y
		.min-segment-size-Y=5.\                             #require min segment size (number of markers) for Chr Y
		hmm-confidence-weight-Y=0.6\                        #parameter for HMM confidence for Chr Y
		.diagonal-weight=0.995.\                            #transition probability matrix diagnal for autosome
		mapd-weight=0.2\                                    #per sample adjustment parameter to HMM state sigmas using MAPD for autosome
		.min-segment-size=5.\                               #require min segment size (number of markers) for autosome
		hmm-confidence-weight=0.6\                          #parameter for HMM confidence for autosome
		.diagonal-weight-X=0.995.\                          #transition probability matrix diagnal for Chr X
		mapd-weight-X=0.2\                                  #per sample adjustment parameter to HMM state sigmas using MAPD for Chr X
		.min-segment-size-X=5.\                             #require min segment size (number of markers) for Chr X, except for the par regions
		hmm-confidence-weight-X=0.6\                        #parameter for HMM confidence for Chr X
		.shrink=true \                                      #true: apply wavelet shrinkage to the log2 ratio before HMM calls; false: otherwise
	--analysis cn-cyto2-gender.\                                #Y-Gender call analysis module
		cutoff=0.5 \                                        #cutoff threshold for Y-Gender call
	--analysis cn-segment \                                     #generate HMM copy number call segment table. Should always specify this for CytoScan750K_Array.
	--analysis lohCytoScan.\                                    #LOH analysis module for CytoScan750K_Array
		lohCS_errorrate=0.05.\                              #control error rate for LOH algorithm
		lohCS_beta=0.001.\                                  #parameter for LOH algorithm
		lohCS_alpha=0.01.\                                  #parameter for LOH algorithm
		lohCS_separation=1000000.\                          #LOH separation parameter for LOH algorithm
		lohCS_nMinMarkers=10.\                              #Minimum marker count for LOH algorithm
		lohCS_NoCallThreshold=0.05.\                        #No call threshold for LOH algorithm
		lohCS_minGenomicSpan=3000000 \                      #Minimum genomic span for LOH algorithm
	--analysis loh-segment \                                    #generate LOH call segment table. Should always specify this for CytoScan750K_Array.
	--analysis cn-neutral-loh \                                 #generate copy neutral LOH segment table. Should always specify this for CytoScan750K_Array.
	--cel-files CELFileList                                     #an ASCII file that contains the full path of input CEL files for single sample mode processing, where the first row of the file is "cel_files" and each subsequent row corresponds to each CEL file
</pre></div> </div>
<hr class="footer"/><address class="footer"><small>Generated for Affymetrix Power Tools by&nbsp;
<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/></a> 1.7.1 </small></address>
</body>
</html>
